### ISSUE TITLE
AI News Digest Shows No Results

### ISSUE THREAD

**Issue by smcgregor** [responsible-ai-collaborative/aiid#2871]
<img width="951" alt="Screenshot 2024-06-15 at 6 02 52 PM" src="https://github.com/responsible-ai-collaborative/aiid/assets/64780/9713b8b5-4e1e-43be-bb94-144256a342ae">

**Comment by kepae** [comment]
We were just discussing how to implement blackbox monitoring/probing of the site for live data features such as this...

@cesarvarela @lmcnulty  -- we were discussing the root issue of this when we saw it transiently, but I can't remember all of the details. The payload size for this function became too large for the AWS lambda serving it? Or for the Netlify function that now carries this in our API?

**Comment by cesarvarela** [comment]
Yes, there are just too many of them, more than 6MB: https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html

**Comment by kepae** [comment]
@lmcnulty 's change mitigated this in production. I'll lower the priority here but keep and re-label this issue to track, because it's important context on the serving infra.
